_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -64,9 +66,9 @@ Both of the above two sections have been focused on increasing flow of business
 Is this a competency required for the duration of the project? If not, a new team member is not the optimal choice, as it is preferable to not interchange members of the team frequently.
 Do not grow the team too large. Ideally we want to keep the agile teams as small as possible, given that the required competencies to deliver business value exist within the team. That means, all things being equal, it is preferable to enhance the skills of a team rather than onboard a new person, if this is an option.
```

Comment:
I would remove the 'we' from the second sentence. "Ideally agile teams should be kept as small as possible..."

Reply:
Added this phrasing